pkg/kn/commands/configuration_edit_flags.go

```diff
@@ -29,6 +29,7 @@ type ConfigurationEditFlags struct {
 	Image                      string
 	Env                        []string
 	RequestsFlags, LimitsFlags ResourceFlags
+	ForceCreate                bool
```

Comment:
i wonder if at some point it would make sense to have 

```
type ConfigurationCreateFlags struct {
  ConfigurationEditFlags
  ForceCreate bool
  ...
}
```

to better separate create from update at the compile time. wdyt?

Reply:
+1
@cppforlife : Do you want me to update the same PR? or log an issue and address is post v0.1 ?

Reply:
@navidshaikh up to you on the timeline. i think current implementation is fine.

Reply:
Logged the suggestion to be addressed later https://github.com/knative/client/issues/110 , thanks @cppforlife 

Also rebased the PR to resolve the conflicts.